fv3core/stencils/dyn_core.py

```diff
@@ -294,6 +302,24 @@ def __init__(
         if self.namelist.rf_fast:
             self._rayleigh_damping = ray_fast.RayleighDamping(self.grid, self.namelist)
 
+        self._compute_pkz_tempadjust = self.initialize_temp_adjust_stencil(
+            self.grid,
+            self._nk_heat_dissipation,
+        )
+
+    @staticmethod
+    def initialize_temp_adjust_stencil(grid, n_adj):
```

Comment:
we can also make this not part of the acoustic object but have it be a free function living in `dyn_core.py`. Do you think that would be better?

Reply:
I'd suggest at minimum making this method private, possibly as a private helper function instead of a class method. You could also consider duplicating this code between the translate class and dyn_core to delete the function altogether.